### PR TITLE
Feature/#6 project progress bar

### PR DESCRIPTION
**Pull request by RouvenGonzalez**
Finished progress bar functionality, application is now fully working as planed.
pls review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,3 +4,4 @@
 - **use the projekt folder for the angular application**
 - **use the androidApp folder for the cordova project**
 
+import { BackendService } from '../services/backend.service';
```
Comment (KlugeCh) [inline README.md]: Line 2, please correct: My Project for the Practical Exam

File: projekt/src/app/progressbar/progressbar.component.ts [hunk 2]
```diff
@@ -1,12 +1,40 @@
...
+    // - 10 tasks finished = 100%
+    // - 5 of 10 tasks finished = 50%
+    // - 1 open , 0 review , 0 finished = 0% : 0/1=0
+    // - 0 open , 1 review , 0 finished = 50% : 0.5/1=0.5
+    // - 0 open , 0 review , 1 finished = 100% :1/1=1
+    // - 0 open , 1 review , 1 finished = 75% : (0.5+1)/2= 0.75
+    // - 1 open , 1 review , 1 finished = 50% : (0+0.5+1)/3= 0.5
+
+    // formel: (status1+status2+...statusN)/n
+    // status : open = 0 , review = 1 , finished = 2
```
Comment (KlugeCh) [inline projekt/src/app/progressbar/progressbar.component.ts]: line 28: You've chosen other values than in : projekt/src/app/models/task.ts
correct: opne = 0, review = 0.5, finished = 1

File: .vscode/launch.json [hunk 3]
```diff
@@ -39,7 +39,23 @@
...
+      ],
+      "console": "integratedTerminal",
+      "internalConsoleOptions": "neverOpen",
+      "disableOptimisticBPs": true,
+      "windows": {
+        "program": "${workspaceFolder}/node_modules/jest/bin/jest"
+      }
     }
   ]
 }
```
Comment (KlugeCh) [inline .vscode/launch.json:61]: only prettier problem : insert return

**Review by KlugeCh — APPROVED** [review]
found only minor problems

File: projekt/src/app/progressbar/progressbar.component.ts [hunk 4]
```diff
@@ -25,7 +25,7 @@ export class ProgressbarComponent implements OnInit {
     // - 1 open , 1 review , 1 finished = 50% : (0+0.5+1)/3= 0.5
 
     // formel: (status1+status2+...statusN)/n
-    // status : open = 0 , review = 1 , finished = 2
+    // status : open = 0 , review = 0.1 , finished = 1
```
Comment (KlugeCh) [inline projekt/src/app/progressbar/progressbar.component.ts]: review = 0.5

**Review by KlugeCh — APPROVED** [review]

File: .vscode/launch.json (hunk 3 above)
Comment (RouvenGonzalez) [reply .vscode/launch.json:61]: I tried to insert one but prettier ereases it on save so im gonna leave it as it is now :)

**Pull request merged**